Fix stalemate condition

christianbundy opened the pull request:
This resolves an issue where stalemates would crash the "recent activity" page. :~)

christianbundy committed:
Fix stalemate condition

Happy0 commented:
Thanks for fixing this, @christianbundy . Would you mind updating the pull request so that there's a separate function for rendering the stalemate message? At the moment, it's under the `renderResignMessage` function.

Maybe we could have a `renderStalemateMessage` function as well?

christianbundy committed:
Run linter

christianbundy committed:
Refactor renderInformation()

This deduplicates code and fixes an issue where the game's state wasn't
output as part of the error text.

christianbundy committed:
Merge branch 'master' of github.com:Happy0/ssb-chess into fix-stalemate-condition

christianbundy commented:
Oops, I suppose I could've *actually read the code*. Sorry about that, I don't know why I thought that would work. Anyway, I've added the above change and run the linter and I think it's ready to go.

Happy0 commented:
Looks good to me. Fanx! =D
